Configuration/Apollo/Apollo-Target-ApolloModels.xcconfig

```diff
@@ -0,0 +1,3 @@
+#include "../Shared/Workspace-Universal-Framework.xcconfig"
+
+INFOPLIST_FILE = Sources/ApolloModels/Info.plist
```

Comment:
My 2 cents: Apollo Android uses `apollo-api` as a name for lib that the generated models uses. I contemplated making this `apollo-runtime`. This is what a lot of other libraries do (antlr, wire, sqldelight, etc...) but there's so much brain muscle around `apollo-api` that I didn't do it eventually.
All in all, I'd personally expect `ApolloModels` to contain models and not "support code for the models". But again, different platform, etc..

Reply:
I like `ApolloRuntime`!

Reply:
i'm not a huge fan of `ApolloRuntime` since I think of the things like networking and dealing with the cache as the runtime, and that's precisely what we're trying **not** to expose here. 

I don't actually have a better suggestion than `ApolloModels`, which has its own problems since not everything in it is precisely a model. 

`ApolloThingsUsedByOurGeneratedCode` seems a hair too long.

Reply:
> I think of the things like networking and dealing with the cache as the runtime

Fair. This is actually the current state of the Android repo. We have `apollo-api` for supporting classes and `apollo-runtime` for networking. 

What about `apollo-client` for networking code etc though? Looks like this is what the web is doing and also that would match the main entry point of the lib?

Reply:
I really like "runtime", but I understand @designatednerd's point here. I'm good with `ApolloAPI` also or `ApolloModelRuntime`. I spent a lot of time this weekend thinking about it, and couldn't come up with anything better. So I just want to make a decision and move on with it at this point honestly. haha

Other ideas I had:

```
ApolloModelEngine
ApolloModelSupport
ApolloAPISupport
ApolloOperationSupport
ApolloGQLOperationSupport
ApolloGQLOperationEngine
ApolloOperationModels
ApolloResponseData
```

Are any of those more appealing to you two?

Reply:
`ApolloAPISupport` or `ApolloModelSupport` both seem flexible but close enough to what we're looking for in my book

Reply:
I'd lean towards `ApolloApi` to match Android but really not sure how that'd sound on iOS so both `ApolloAPISupport` or `ApolloModelSupport` work for me!